src/app/public/plugin-resources/code-examples/page-summary/basic/page-summary-demo.component.html

```diff
@@ -0,0 +1,125 @@
+<div
+  style="border: solid 1px #ccc"
```

Comment:
Since these demos are supposed to be "copy-paste-worthy", maybe we should use the `sky-border-light` CSS class for this? Found here: https://developer.blackbaud.com/skyux/design/styles/borders

Reply:
(Or just remove it? Do we need the border?)

Reply:
@Blackbaud-ToddRoberts do you know why we added this border? You can see it in the current demo as well: https://developer.blackbaud.com/skyux/components/page-summary#demo
Can I remove it?

Reply:
To set the page summary elements apart from the surroundings and show the built in padding. Can use the CSS class instead of hard coding.

Reply:
Talked to Todd offline. The best move here is to remove it from the demo since we don't want consumers copy+pasting.